examples/static-react/src/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This example uses Express to serve static files
+ */
+
+const express = require( 'express' );
+const path = require( 'path' );
+
+const app = express();
+
+const PORT = process.env.PORT || 3000;
+const BASEURL = process.env.BASEURL || 'http://localhost/';
```

Comment:
so that it prints correctly when combined with the PORT
```suggestion
const BASEURL = process.env.BASEURL || 'http://localhost';
```